sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2657,15 +2650,15 @@
           "type": "string"
         },
         "outputBlobContainerUri": {
-          "description": "URI containing SAS token to a blob container.  This is used to output the status of the job and the results.",
+          "description": "URI containing SAS token to a blob container. This is used to output the status of the job and the results.",
           "type": "string"
         },
         "outputBlobName": {
-          "description": "The name of the blob that will be created in the provided output blob container.  This blob will contain\r\nthe exported device registry information for the IoT Hub.",
+          "description": "The name of the blob that will be created in the provided output blob container. This blob will contain the exported device registry information for the IoT Hub.",
```

Comment:
I'm not sure I understand this the first sentence they originally had. Is it not sufficient to say "The name of the blob created by the job"? Followed by the explanation you have.

Reply:
I am guessing they have it this way because one of the properties is also outputBlobContainerUri. So they want to call explicitly which container the blob was created on.

We need a content editor expert to help us with this.